setup.cfg

```diff
@@ -12,17 +12,3 @@ create-wheel = yes
 extend-ignore = E501
 # https://pep8.readthedocs.org/en/latest/intro.html#error-codes
 # E501: line too long (82 > 79 characters)
```

Comment:
Other than `[bdist_wheel] universal = 1` that can be removed, the `[metadata]` section has an old spelling of the option that produces 

> /usr/lib/python3/dist-packages/setuptools/config/setupcfg.py:515: SetuptoolsDeprecationWarning: The license_file parameter is deprecated, use license_files instead.
  warnings.warn(msg, warning_class)

whenever I run `python3 setup.py sdist` or `python3 setup.py bdist_wheel`.

I wonder, is there really no equivalent of that option in pyproject.toml?

I originally added that snippet to my setup.cfg because some blog post said it was The Right Thing To Do to make sure binary wheel files do not violate the license by including the license text alongside the code.

Now I'm looking at https://setuptools.pypa.io/en/latest/references/keywords.html#keywords and it seems it could've gone into setup.py, alongside the license = 'GPL' bit, and also it looks like the default value works fine for me, so it can be removed altogether.

Test plan: run `python3 setup.py bdist_wheel` or `python3 -m build` and make sure dist/*.whl contains the LICENSE file.

Well, I edited setup.cfg, removed the `[bdist_wheel]` and `[metadata]` sections, build the wheel using both methods, and saw that the LICENSE file was present in the *.dist-info/ subdirectory inside the wheel.

Verdict: just remove the entire setup.cfg.  Oh, the `[flake8]` config, right, that can maybe go into a new `.flake8` file.  (One of these days I will look for a linter that is not flake8.)